src/SSHLibrary/abstractclient.py

```diff
@@ -1109,6 +1109,8 @@ def _get_put_file_sources(self, source):
         return sources
 
     def _get_put_file_destinations(self, sources, destination, path_separator):
+        if destination[1:3] == ':' + os.sep:
+            destination = os.sep + destination
```

Comment:
Should you use path_separator instead of os.sep here? os.sep is separator of the host system, not destination.

Reply:
Thanks for the heads up! Fixed it.